cvat/apps/annotation/README.md

```diff
@@ -150,3 +150,173 @@ It allows to download and upload annotations in different formats and easily add
+This is native CVAT annotation format.
+[Detailed format description](cvat/apps/documentation/xml_format.md)
+
+##### CVAT XML for images
+downloaded file: Single unpacked XML
+supported shapes - Rectangles, Polygons, Polylines, Points
+
+##### CVAT XML for videos
+downloaded file: Single unpacked XML
+supported shapes - Rectangles, Polygons, Polylines, Points
+
+#### CVAT XML Loader
+uploaded file: Single unpacked XML
+supported shapes - Rectangles, Polygons, Polylines, Points
+
+
+### [Pascal VOC](http://host.robots.ox.ac.uk/pascal/VOC/)
+
+#### Dumper description
+- downloaded file: a zip archive with following structure:
+  ```
+  taskname.zip
+  ├── frame000001.xml
+  ├── frame000002.xml
+  ├── frame000003.xml
+  └── ...
+  ```
+  Each *.xml file contains annotations for a frame with the same name as annotation file. Detailed structure specification of the *.xml file can be found [here](http://host.robots.ox.ac.uk/pascal/VOC/voc2012/devkit_doc.pdf)
+- supported shapes - Rectangles
+- additional comments: If you plan to use 'truncated' and 'difficult' attributes please add to CVAT labels attributes the corresponding items:
+  `~checkbox=difficult:false ~checkbox=truncated:false`
+
+#### Loader description
+- uploaded file: a zip archive with following structure:
+  ```
+  taskname.zip
+  ├── frame000001.xml
+  ├── frame000002.xml
+  ├── frame000003.xml
+  └── ...
+  ```
+  Each *.xml file contains annotations for a frame with the same name as annotation file. Note: it should be possible to match the CVAT frame(imagename) and image filename from the annotation *.xml file(tag filename, e.g. `<filename>2008_004457.jpg</filename>`). There are 2 options:
+  1. full match between image name and filename form annotation *.xml file (in case of a task was created from images or archive of images).
+  2. match by frame number (if CVAT cannot match by name). If filename tag contains a number CVAT will interpret it as frame number.
+
+- supported shapes: Rectangles
+- limitations: Support of Pascal VOC object detection format
+- additional comments: the CVAT task should be created with the full label set that may be in the annotation files
+
+#### How to create a task from Pascal VOC dataset
+1. Download the Pascal Voc dataset
+1. Create the CVAT task with the following labels:
+   ```
+   aeroplane bicycle bird boat bottle bus car cat chair cow diningtable dog horse motorbike person pottedplant sheep sofa train tvmonitor
+   ```
+   You can add `~checkbox=difficult:false ~checkbox=truncated:false` attributes if you want to use it.
+
+   Select interesting image files (See [Creating an annotation task](cvat/apps/documentation/user_guide.md#creating-an-annotation-task) guide for details)
+1. zip the corresponding annotation files
+1. click `Upload annotation` button, choose `Pascal VOC ZIP 1.0` and select the *.zip file with annotations from previous step. It may take some time.
+
+### [YOLO](https://pjreddie.com/darknet/yolo/)
+#### Dumper description
```

Comment:
Need an example for YOLO as well. For example, you can redirect the dataset preparation to https://pjreddie.com/darknet/yolo/ article.